### DISCUSSION TITLE
How does the auction order flow work?

### DISCUSSION THREAD

**Discussion by JasperAlexander** [ProjectOpenSea/seaport#552] [Q&A]
Can someone describe the auction order flow when using an off-chain order book? What should you save in an off-chain order book and where in the order flow? For example:

Seller: mints ERC721 > approves ERC721 to Seaport contract > creates order 1 > save order 1 to off-chain order book(???)
Buyer: mints ERC20 > approves ERC20 to Seaport contract > creates mirror of order 1
Seller: fulfills order 1 from off-chain order book(???)

**Comment by ryanio** [comment]
You can set the zone to an EOA who can programmatically execute with the winner of the auction. The seller can also accept an offer from the highest bidder from the off chain order book.
